.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: go
+
+services:
+  - mongodb
+
+go:
+  - 1.7.1
+  - tip
+
+before_script:
+  - go get -u github.com/golang/lint/golint
+  - for package in $(go list ./... | grep -v vendor | uniq); do golint -set_exit_status $package; done
+  - go vet $(go list ./... | grep -v vendor | uniq)
+
+script:
+  - go test $(go list ./... | grep -v '/vendor/')
```

Comment:
this grep uses `'/vendor/'` but the others just use `vendor`. I like the explicitness of this one so maybe make the others match?